docs/kintsugi/Earn-With-kBTC.md

```diff
@@ -0,0 +1,26 @@
+# Earn With Your kBTC
+
+Since one of the main reasons for issuing kBTC is to use it with DeFi dApps or protocols, we have collected below some of the use cases and opportunities that you can consider. 
+
+Please note that while we are happy to surface such use cases and might provide incentives for one or more, this is by no means financial advice. Please do your own research and inform yourself about impermanent loss and about each dApp's or protocol's policies and fees.   
+
+### All Opportunities At A Glance:
+
+[![Solarbeam Pool](../_assets/img/kintsugi/Solarbeam%20kBTC%20Pool.png)](https://app.solarbeam.io/farm?filter=stable) 
```

Comment:
@alexbleau you need to add the images to `/docs/_assets/img/kintsugi/..`. I can also quickly do it if you send me the images.